src/atoms/Typography/Typography.tsx

```diff
@@ -4,7 +4,18 @@ import styled from '../../styled-components';
 export const Typography = styled.p`
   color: ${props => props.theme.text};
   font-family: Lato, sans-serif;
+  font-weight: bold;
```

Comment:
I think this should be inside `a {}`, it's making all Typography children bold.

Reply:
Good call!